<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>170</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>170</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Chose ticket</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>170</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>My tickets</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>170</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Feedback</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>If you have any questions or offers </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>You can write to:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mail : &lt;a href=&quot;podgurskaja01@gmail.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;podgurskaja01@gmail.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Telegram : &lt;a href=&quot;http://t.me/ana_stasya00&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;http://t.me/ana_stasya00&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
